packages/fast-usdc/test/cli/cli.test.ts

```diff
@@ -105,19 +105,115 @@ test('shows help for config show command', async t => {
   t.snapshot(output);
 });
 
+test('shows help for deposit command', async t => {
+  const output = await collectStdOut([CLI_PATH, 'deposit', '-h']);
+
+  t.snapshot(output);
+});
+
+test('shows help for withdraw command', async t => {
+  const output = await collectStdOut([CLI_PATH, 'withdraw', '-h']);
+
+  t.snapshot(output);
+});
+
+test('shows error when deposit command is run without options', async t => {
+  const output = await collectStdErr([CLI_PATH, 'deposit']);
+
+  t.snapshot(output);
+});
+
+test('shows error when deposit command is run with invalid amount', async t => {
+  const output = await collectStdErr([CLI_PATH, 'deposit', 'not-a-number']);
+
+  t.snapshot(output);
+});
+
+test('shows error when deposit command is run with invalid fee', async t => {
+  const output = await collectStdErr([
+    CLI_PATH,
+    'deposit',
+    '50',
+    '--fee',
+    'not-a-number',
+  ]);
```

Comment:
To make these look more like command lines, consider...

```suggestion
  const output = await collectStdErr([CLI_PATH, ...'deposit 50 --fee not-a-number'.split(' ')]);
```

Reply:
Not really my preference, though I see where you're coming from.